src/main/java/pages/opportunity/OpportunityForm.java

```diff
@@ -0,0 +1,196 @@
+package pages.opportunity;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.CacheLookup;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import pages.LookupPage;
+import pages.basepages.FormBase;
+
+import java.util.LinkedList;
+import java.util.Set;
+
+/**
+ * Created by Virginia Sanabria on 8/30/2015.
+ */
+public class OpportunityForm extends FormBase{
+    @FindBy(id = "opp3")
+    @CacheLookup
+    private WebElement opyNameTxt;
+
+    @FindBy(id = "opp5")
+    @CacheLookup
+    private WebElement typeComBox;
+
+    @FindBy(id = "opp6")
+    @CacheLookup
+    private WebElement leadSourceComBox;
+
+    @FindBy(id = "opp7")
+    @CacheLookup
+    private WebElement amountTxt;
+
+    @FindBy(css = "img[title *= 'Account Name Lookup']")
+    @CacheLookup
+    private WebElement lookupIcon;
+
+    @FindBy(id = "opp9")
+    @CacheLookup
+    private WebElement closeDateTxt;
+
+    @FindBy(id = "opp10")
+    @CacheLookup
+    private WebElement nextStepTxt;
+
+    @FindBy(id = "opp11")
+    @CacheLookup
+    private WebElement stageComBox;
+
+    @FindBy(id = "opp12")
+    @CacheLookup
+    private WebElement probabilityTxt;
+
+    @FindBy(xpath= "//td[contains(., 'Order Number')]/following-sibling::td/input")
```

Comment:
We need improve this selector because currently it is matching two components.

![image](https://cloud.githubusercontent.com/assets/20162/9603155/b7a51994-507c-11e5-829f-4a916435812e.png)